crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -1286,6 +1305,12 @@ impl JsValue {
                 "!({})",
                 value.explain_internal_inner(hints, indent_depth, depth, unknown_depth)
             ),
+            JsValue::Iterated(_, iterable) => {
+                format!(
+                    "for (let _ of {})",
```

Comment:
```suggestion
                    "Iterated({})",
```

Should read like an expression